_includes/head.html

```diff
@@ -4,7 +4,14 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
 
     <title>{% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}{% if page.tagline %} || {{page.tagline}}{%endif%} || OpenTechSchool.org</title>
-    <meta name="description" content="{{ site.description }}">
+    <meta name="description" content="{{ site.description }}" />
+	  <meta name="author" content="OpenTechSchool">
```

Comment:
Yeah would be nicer if it has too.  `/` is semantically correct even if most of the modern browsers will fix it for us.
I would say is not a blocker but I will add it in the future for sure.